vkcs/internal/generate/providerschema/main.go

```diff
@@ -52,7 +51,7 @@ func (g ProviderSchemaJSONGenerator) Code() string {
 }
 
 func (g ProviderSchemaJSONGenerator) codeForSchema() string {
-	pSchemaJSON, err := ioutil.ReadFile(g.SchemaPath)
+	pSchemaJSON, err := os.ReadFile(g.SchemaPath)
```

Comment:
why this has been changed?

Reply:
Golangci-lint show me this line, because ioutil is deprecated